pages/common/glab-auth.md

```diff
@@ -0,0 +1,20 @@
+# glab auth
+
+> Authenticate with a GitLab host from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/auth/index.html#synopsis>.
+
+- Log in with interactive prompt:
+
+`glab auth login`
+
+- Log in with a token from standard input (created in https://gitlab.com/-/profile/personal_access_tokens):
+
+`echo {{your_token}} | glab auth login --stdin`
+
+- Check if you are logged in:
```

Comment:
probably this sounds better (thoughts ?):

```suggestion
- Check authentication status:
```

Reply:
We try to avoid `you`/`your`.
```suggestion
- Check the authentication status:
```